src/server/modes/charts/plugins/control/js/index.ts

```diff
@@ -112,6 +112,7 @@ export default ({
         multiselect: source.multiselectable,
         content,
         param,
+        isValueRequired: source.isValueRequired,
```

Comment:
TBH `Value` is excessive for Control. `IsRequired` would be better or even just `required`